### PR TITLE
Default self service user attrs to Unknown

### PR DESCRIPTION
**Pull request by mango-habanero**

### CODE REVIEW

File: app/server/utils/user.py [hunk 1]
```diff
@@ -843,7 +843,7 @@ def create_user_without_transfer_account(phone):
     :return: User
     """
     temporary_first_name = 'Unknown'
-    temporary_last_name = ''
+    temporary_last_name = 'Unknown'
```
Comment (nolash) [inline app/server/utils/user.py]: Perhaps a different name so we can distinguish first/last? Feel free to be creative :)

File: app/server/utils/user.py [hunk 2]
```diff
@@ -880,6 +879,9 @@ def attach_transfer_account_to_user(user):
     blockchain_address = bt.create_blockchain_wallet()
     user.primary_blockchain_address = blockchain_address
     user.set_held_role('BENEFICIARY', 'beneficiary')
+    temporary_gender = 'Unknown'
```
Comment (nolash) [inline app/server/utils/user.py]: same here. 

perhaps "Unknown gender" "Unknown business" etc

File: app/server/utils/ussd/kenya_ussd_state_machine.py [hunk 3]
```diff
@@ -385,27 +385,31 @@ def is_ussd_signup(self, user_input):
...
-        if not self.user.location:
+        if not self.user.location or self.user.location == 'Unknown':
             return True
         else:
             return False
 
     def has_empty_bio_info(self, user_input):
         bio = next(filter(lambda x: x.name == 'bio', self.user.custom_attributes), None)
-        if not bio:
+        print(bio)
```
Comment (nolash) [inline app/server/utils/ussd/kenya_ussd_state_machine.py]: Should this print be here?

File: app/server/utils/user.py [hunk 4]
```diff
@@ -842,14 +843,14 @@ def create_user_without_transfer_account(phone):
...
-    temporary_last_name = 'Unknown'
+    temporary_first_name = 'Unknown first name'
+    temporary_last_name = 'Unknown last name'
     user = User(first_name=temporary_first_name,
                 last_name=temporary_last_name,
                 phone=phone,
                 registration_method=RegistrationMethodEnum.USSD_SIGNUP)
 
-    organisation = g.active_organisation
+    organisation = Organisation.query.get(2)
```
Comment (nolash) [inline app/server/utils/user.py]: ohh I'm not sure about hardcoding the `2`

**Review by nolash — CHANGES_REQUESTED** [review]

File: app/server/utils/user.py (hunk 4 above)
Comment (mango-habanero) [reply app/server/utils/user.py]: I've reverted this in this: [Reverts organization to which user's are assigned](https://github.com/GrassrootsEconomics/SempoBlockchain/pull/35/commits/40e86ca6f9e5b0aef048402f99cd4d7028347a0e)

**Review by nolash — APPROVED** [review]

**Pull request merged**